src/central-submissions.rst

```diff
@@ -104,13 +104,27 @@ Learn more about these options below:
 Downloading submissions as CSVs
 -------------------------------
 
-To download all submission data as a :file:`.zip` of :file:`.csv` tables, click on the :guilabel:`Download all # records` button on the right side of the listing page:
+To download all submission data as :file:`.csv` tables, click on the :guilabel:`Download all # records` button on the right side of the listing page:
 
    .. image:: /img/central-submissions/download-button.png
 
 If you have any row filters applied to the submission table, those filters will be applied to your download as well. You can use this to, for example, download only submissions from a particular month, or only approved submissions.
 
-Once the :file:`.zip` completes downloading, you will find one or more files when you extract it:
+Once the download dialog opens, you'll be given some additional export options.
+
+   .. image:: /img/central-submissions/download-modal.png
+
+Some of the options may be disabled if they do not apply to your data, or if they are not available due to features that you have enabled (such as managed encryption).
+
+The first two options help make Central's data output match Briefcase's more closely. If you are migrating from Briefcase, consider using them:
```

Comment:
I think these two sentences can be dropped. These were options in Briefcase as well. They exist because they're common requests -- users needed them so badly that they still used Briefcase.

What we should do is link to this section from the Briefcase docs but we can do that separately.